backend/tests/reservations.test.js

```diff
@@ -0,0 +1,58 @@
+const db = require('../src/db');
+const utils = require('../src/utils');
+
+describe('Reservation Testing', () => {
+    let reservation = {
+        id: 1,
+        courseId: 186399,
+        eventId: 2554743,
+        userId: 123,
+        ticketType: 'Single_Ticket',
+        ticketPrice: 90.5,
+        bookingStatus: 0,
+    };
+
+    test('Should create reservations', async () => {
+        await db.reservations.createReservation(reservation);
+        const dbreservation = await db.reservations.getReservationById(
+            reservation.id
+        );
+        expect(dbreservation.id).toEqual(reservation.id);
+        expect(dbreservation.userId).toEqual(reservation.userId);
+        expect(dbreservation.ticketPrice).toEqual(reservation.ticketPrice);
+        expect(dbreservation.bookingStatus).toEqual(reservation.bookingStatus);
+    });
+
+    test('Should validate Booking Status', async () => {
```

Comment:
This is not `async` test, please make sure not to use async where it is not necessary.